github/event_types.go

```diff
@@ -265,7 +265,7 @@ type DeploymentProtectionRuleEvent struct {
 	Environment *string `json:"environment,omitempty"`
 	Event       *string `json:"event,omitempty"`
 
-	// The URL Github provides for a third-party to use in order to pass/fail a deployment gate
+	// The URL GitHub provides for a third-party to use in order to pass/fail a deployment gate
```

Comment:
I know that most instances of `Github` are intentional, but I claim that this instance doesn't need to be spelled according to golang conventions.